lib/vault/encrypted_model.rb

```diff
@@ -41,31 +42,22 @@ module ClassMethods
       #   a proc to encode the value with
       # @option options [Proc] :decode
       #   a proc to decode the value with
+      # @option options [Hash] :transform_secret
+      #   a hash providing details about a transformation to use,
+      #   or a name of an existing transformation
```

Comment:
Since this no longer accepts a string, is it safe to remove “or a name of an existing transformation”?

Reply:
Or s/or a/such as the/